File: 	1041.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [x, y] = lines.shift().split(" ");


if(x==0.0 && y==0.0){
console.log('Origem');
}

else if(x<0.0 && y<0.0){
    console.log('Q3');
}

 else if(x>0.0 && y<0.0){
    console.log('Q4');
}

else if(x<0.0 && y>0.0){
    console.log('Q2');
}

else if(x>0.0 && y>0.0){
    console.log('Q1');
}

 else if(x==0.0 && y!=0.0){
    console.log('Eixo Y');
}

else{
    console.log('Eixo X');
}

File: 	1041.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		double X = sc.nextDouble();
		double Y = sc.nextDouble();

		if (X > 0.0 && Y < 0.0) {
			System.out.println("Q4");

		}
		if (X > 0.0 && Y > 0.0) {
			System.out.println("Q1");
		}
		if (X < 0.0 && Y > 0.0) {
			System.out.println("Q2");
		}
		if (X < 0.0 && Y < 0.0) {
			System.out.println("Q3");
		}
		if (X == 0.0 && Y == 0.0) {
			System.out.println("Origem");
		}
		if (X == 0.0 && Y != 0.0) {
			System.out.println("Eixo Y");
		}
		if (X != 0.0 && Y == 0.0) {
			System.out.println("Eixo X");
		}
	}
}
